### DISCUSSION TITLE
Viewing cached packages through client

### DISCUSSION THREAD

**Discussion by jshen28** [devpi/devpi#937] [Q&A] [answered]
Hi,

   Is it possible to see all cached packages through client? I saw the threads talking about using web UI, but curious to know if it is
possible through reset API. 
   Thank you very much for the help.

Best,
Jiatong shen

**Comment by fschulze** [comment, marked as answer]
Not yet. What is your use-case for that?

**Reply by jshen28** [reply]
Thank you for reply. My colleague asks me how many packages have been cached. Maybe their use case is for auditing.
